lib/ansible/cli/galaxy.py

```diff
@@ -12,7 +12,6 @@
 import sys
 import textwrap
 import time
-
```

Comment:
This empty line was a separator between stdlib imports and the third-party ones. Please revert any formatting changes as they are unrelated to what you claim to patch in the PR title.